libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -1072,6 +1073,9 @@ class MapboxNavigation(
         )
     }
 
+    private fun getHistoryPath() =
```

Comment:
This should be public, but it's really nice if it's in the `MapboxHistoryRecorder`.